src/main.js

```diff
@@ -129,21 +128,38 @@ function mouseMoveListener(e) {
  * - existing link: 
  */
 function singleClickEvent(e) {
+  if(mouseUp != 1 || mouseDown != 1){
+    return;
+  }
```

Comment:
It's better to move this check up to the event listener. Only trigger this even when mouseUp = 1 and mouseDown = 1.

Reply:
removed this functionality, and instead reset timers in resetState()